assets/src/stories-editor/components/right-click-menu/index.js

```diff
@@ -100,16 +208,53 @@ const RightClickMenu = ( props ) => {
 				{
 					name: __( 'Duplicate Block', 'amp' ),
 					blockAction: duplicateBlock,
+					params: [ firstBlockClientId ],
 					icon: 'admin-page',
 					className: 'right-click-duplicate',
 				},
 			);
 		}
 
+		const pageList = getBlockOrder();
+		const pageNumber = pageList.length;
```

Comment:
What about `numPages` or `totalPages` or something? Was a bit confusing at first.

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3417/commits/558739e2b25c4870320736e30689e719345b64f5